app/src/locales/en.json

```diff
@@ -118,6 +118,15 @@
   "retry": "Retry",
   "mergingScript": "Merging script",
   "selectScript": "Select script",
+  "runNewBatch": "Run a new batch",
+  "selectResources": "Select resources",
+  "selectAllResources": "Select all resources",
+  "unselectResources": "Unselect resources",
+  "runOn": "Run on",
```

Comment:
```suggestion
  "runOnResources": "Run on {{resourceCount}} resources",
```

Reply:
You can also handle plural : https://www.i18next.com/translation-function/plurals

If you plan on using this feature, take in mind that instead of naming the count variable `resourceCount`, it will have to be `count`